kornia/augmentation/_2d/__init__.py

```diff
@@ -0,0 +1,3 @@
+from kornia.augmentation._2d.geometric import *
```

Comment:
why `_2d` and not `2d` ?

Reply:
throws invalid syntax error. IIRC, python package names must start with a literal or underscore

Reply:
just out of curiosity, could you share link to that ?

Reply:
[good ol' stackoverflow](https://stackoverflow.com/a/17487228) with valid links to [identifiers](https://docs.python.org/3/reference/lexical_analysis.html#identifiers) and [grammar-token-module](https://docs.python.org/3/reference/simple_stmts.html#grammar-token-module) for Python 3